### ISSUE TITLE
Vue3 - no visible calendar, CSS doesn't seem to be importing

### ISSUE THREAD

**Issue by mdeblieck** [Vuepic/vue3-date-time-picker#36]
**Describe the bug**
Installed component in a Vue3 app, imported it locally along with the CSS. Component appears on page, however the CSS does not appear to be importing. No errors reported. See attached image - the left is before clicking to open, the right is when you click to open. No calendar appears.

The CSS file exists in the node_modules folder

To **reproduce**
Simply installed component, imported and added to existing page. Cannot find any cause as to why it's not appearing properly.

**Comment by Jasenkoo** [comment]
Possibly related to (#35).

Can you install `v2.3.1` and let me know if you still have the problem.

**Comment by mdeblieck** [comment]
Unfortunately I had already tried that. I also tried downgrading to 2.3.0, and 2.2.0, but no difference.

**Comment by Jasenkoo** [comment]
Are you using some CSS framework? What is the Vue version installed? Can you show me some code snippets or where do you import code?

**Comment by mdeblieck** [comment]
Using Laravel, Vue 3.2.6, InertiaJS and Tailwind CSS.
Image attached with some snippets.
![datepicker2](https://user-images.githubusercontent.com/89146550/139948883-829ed6e0-5c37-4ef4-be7e-d1ece2ccb86b.png)

**Comment by Jasenkoo** [comment]
Ok, tnx, I will take a deeper look

**Comment by Jasenkoo** [comment]
Temp fix

In the component add a style tag with import

```vue
<style lang="css">
    @import "~/node_modules/vue3-date-time-picker/dist/main.css";
</style>
```

**Comment by mdeblieck** [comment]
That works, thanks!

**Comment by Jasenkoo** [comment]
The issue is fixed in the new version [2.3.3](https://github.com/Vuepic/vue3-date-time-picker/releases/tag/v2.3.3)
